apps/fm/js/fm.js

```diff
@@ -184,11 +184,24 @@ function updatePowerUI() {
 }
 
 function updateAntennaUI() {
-  $('antenna-warning').hidden = mozFMRadio.antennaAvailable;
```

Comment:
Good point. What about unifying both methods into one called something like updateWarningModeUI():
we would then do something like:
```
$('airplane-mode-warning').hidden = !airplaneModeEnabled;
$('antenna-warning').hidden = airplaneModeEnabled || mozFMRadio.antennaAvailable;
$('container').hidden = airplaneModeEnabled || !mozFMRadio.antennaAvailable;
```